apps/src/lab2/views/components/versionHistory/VersionHistoryDropdown.tsx

```diff
@@ -309,129 +310,136 @@ const VersionHistoryDropdown: React.FunctionComponent<
   }, [closeDropdown, dispatch, isLatestVersion, selectedVersion]);
 
   return createPortal(
-    <div
-      className={moduleStyles.versionHistoryDropdown}
-      ref={menuRef}
-      role="dialog"
-      style={dropdownStyles}
-      aria-modal="true"
-      aria-label={lab2I18n.versionHistoryList()}
+    <FocusTrap
```

Comment:
supernit: Non blocking, no need to update, just wanted to share info that we also have useFocusTrap hook in component-library that can also be used if needed, it works on refs. Though it's little bit less complex then 'focus-trap-react'

[frontend/packages/component-library/src/common/hooks/useFocusTrap.ts](https://github.com/code-dot-org/code-dot-org/blob/b93ceae9a3f0df678cbd79d6c27109827ac37281/frontend/packages/component-library/src/common/hooks/useFocusTrap.ts)

Example usage can be found here:  https://github.com/code-dot-org/code-dot-org/blob/b93ceae9a3f0df678cbd79d6c27109827ac37281/frontend/packages/component-library/src/dialog/CustomDialog.tsx#L50-L52

Reply:
oh nice okay! I didn't know about this and will look into it next time thanks!